src/shared/logger/commands.ts

```diff
@@ -30,10 +30,11 @@ export class Logging {
 
     public constructor(private readonly defaultLogUri: vscode.Uri, private readonly logger: Logger) {}
 
-    public async openLogUri(logUri = this.defaultLogUri): Promise<vscode.TextEditor | undefined> {
+    public async openLogUri(logUri?: unknown): Promise<vscode.TextEditor | undefined> {
         telemetry.toolkit_viewLogs.emit() // Perhaps add additional argument to know which log was viewed?
 
-        return vscode.window.showTextDocument(logUri)
+        // If the command is called directly from a package.json menu item, then logUri may be a vscode-provided object, typically a TreeNode.
+        return vscode.window.showTextDocument(logUri instanceof vscode.Uri ? logUri : this.defaultLogUri)
```

Comment:
IIUC, tree nodes might drop a different parameter in? Are we always expecting it to open `this.defaultLogUri`? If so, can we just set that and remove the parameter entirely?

Reply:
Code doesn't explicitly use another parameter anywhere, but it seemed implicitly used via some javascript magic/vscode context. Was unable to repro a feature that used another log file. Removing for now, since it is unused.